Explorer/Assets/Scripts/SceneRunner/ECSWorld/ECSWorldFactory.cs

```diff
@@ -14,15 +17,21 @@ public ECSWorldFactory(IComponentPoolsRegistry componentPoolsRegistry /* Add her
             this.componentPoolsRegistry = componentPoolsRegistry;
         }
 
-        public ECSWorldFacade CreateWorld()
+        public ECSWorldFacade CreateWorld(Dictionary<CRDTEntity, Entity> entitiesMap)
         {
             // Worlds uses Pooled Collections under the hood so the memory impact is minimized
             var world = World.Create();
 
+            // We create the scene root transform
+            var sceneRootTransform = (Transform)componentPoolsRegistry.GetReferenceTypePool(typeof(Transform)).Rent();
+            sceneRootTransform.name = "SCENE_ROOT";
```

Comment:
We should be giving some valuable names for different scenes, I propose to move the logic for Transform creation to the outer layer, handle it there, and add support for recycling these transforms on `Dispose`, otherwise they will live forever every time the scene is [re]launched